net/release-notes/release-notes-2021/aspose-pdf-for-net-21-12-release-notes/_index.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Aspose.PDF for .NET 21.12 Release Notes
+linktitle: Aspose.PDF for .NET 21.12
+type: docs
+weight: 70
+url: /net/aspose-pdf-for-net-21-12-release-notes/
+lastmod: "2021-12-15"
+description: This page contains new Aspose.PDF for .NET features, enhancement, and bug fixes in 2021, version 21.12. 
+sitemap:
+    changefreq: "weekly"
+    priority: 0.7
+---
+
+{{% alert color="primary" %}}
+
+This page contains release notes information for Aspose.PDF for .NET 21.12.
+
+{{% /alert %}} 
+
+## Improvements and Changes
+
+|**Key**|**Summary**|**Category**|
+| :- | :- | :- |
+|PDFNET-50991|Create example fixup|Feature|
+|PDFNET-46044|Increase performance while adding text|Enhancement|
+|PDFNET-40008|IFF to PDF - Exception during conversion|Bug|
+|PDFNET-45059|F to JPG - Cross reference table or cross reference stream not found|Bug|
```

Comment:
|PDFNET-45059|PDFF to JPG - Cross-reference table or cross-reference stream not found|Bug|